browse/templates/tb/tb.html

```diff
@@ -38,5 +51,5 @@ <h2>Trackbacks for <a href="{{ url_for('.abstract', arxiv_id=arxiv_identifier.id
   {% else %}
   <p><em>There are no trackback pings recorded for <a href="{{ url_for('.abstract', arxiv_id=arxiv_identifier.id) }}">{{ arxiv_identifier.id }}</a>.</em></p>
   {% endif %}
-  {{ super() }}
```

Comment:
The TB page looks good.
I just wanted to check that this is an intentional removal of this call to the parent template.

Reply:
it is intentional. That call was just moved to a different containing block for layout reasons.